docs/install/declarative-install.md

```diff
@@ -122,23 +122,52 @@ This guide will demonstrate how to install Ondat onto a Kubernetes cluster decla
     kubectl get storageclasses | grep "storageos"
     ```
 
-### Option B - Using Ondat's Operator Helm Chart
+### Option B - Using Ondat's Helm Chart
 
 #### Step 1 - Install Helm
 
 * Ensure that the [Helm 3](https://helm.sh/) CLI utility is installed on your local machine and is available in your `$PATH`:
   * [helm](https://helm.sh/docs/intro/install/)
 
-#### Step 2 - Setup An `etcd` Cluster
+#### Step 2 - Setup An `etcd` Cluster (External etcd)
 
-* Ensure that you have an `etcd` cluster deployed first before installing Ondat through the Helm chart. There are two different methods listed below with instructions on how to deploy an `etcd` cluster;
+* If you are installing `etcd` externally, ensure that you have deployed the cluster before installing Ondat through the Helm chart. There are two different methods listed below with instructions on how to deploy an `etcd` cluster;
 
     1. [Embedded Deployment](https://docs.ondat.io/docs/prerequisites/etcd/#testing---installing-etcd-into-your-kubernetes-cluster) - deploy an `etcd` cluster operator into your Kubernetes cluster, recommended for **non production** environments.
     1. [External Deployment](https://docs.ondat.io/docs/prerequisites/etcd/#production---etcd-on-external-virtual-machines) - deploy an `etcd` cluster in dedicated virtual machines, recommended for **production** environments.
 
 * Once you have an `etcd` cluster up and running, ensure that you note down the list of `etcd` endpoints as comma-separated values that will be used when configuring Ondat in **Step 4**.
   * For example, `203.0.113.10:2379,203.0.113.11:2379,203.0.113.12:2379`
 
+#### Step 2 - Setup a `StorageClass` for `etcd` (Internal etcd)
+
+* If you are installing `etcd` inside the cluster, ensure that you have at least 3 (recommend 5) nodes ready to ensure high availability. It is recommended that these nodes are placed in different physical or virtual locations (ie. Datacenters or availability zones) for maximum resilience.
+
+* Before installing Ondat with `etcd`, create the `StorageClass` that you want to use for `etcd`. Note that this cannot be `storageos` as Ondat depends on etcd to function. The following procedure will install a local path `StorageClass` that will work in all configurations, ideally there is another more resilient option (eg. Gp3 on AWS) available that can be used instead.
+
+1. By default, a newly provisioned Kubernetes cluster does not have any CSI driver deployed. Run the following commands against the cluster to deploy a [Local Path Provisioner](https://github.com/rancher/local-path-provisioner) to provide local storage for Ondat's embedded `etcd` cluster operator deployment.
```

Comment:
I don't think we should be so prominently displaying how to use a non-production ready storage class.

We should at least explain how to setup a production environment (i.e how to setup an EBS csi driver)